src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputData) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : inputData) {
+            if (line.startsWith("type")) {
+                continue;
+            }
```

Comment:
This code will skip the first line only if it starts with the word 'type'. But what if the first line is formatted differently or the file doesn't have a header at all? It's better to handle the header separately before processing the rest of the lines. This will make your code more flexible and less prone to errors.

Reply:
changes have been made